db/migrate/20210215153212_add_code_region_to_solicitations.rb

```diff
@@ -0,0 +1,8 @@
+class AddCodeRegionToSolicitations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :solicitations, :code_region, :integer
+    add_index :solicitations, :code_region
+    # Tache `rake add_code_region_to_solicitations` a lancer ensuite
```

Comment:
```suggestion
    # Tache `rake update_solicitations_code_region` a lancer ensuite
```
Je ne sais pas si c'est ça que tu voulais dire, je n'ai trouvé nulle part 'add_code_region_to_solicitations'